modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -655,12 +655,18 @@
 			});
 		},
 
+		/**
+		 * @deprecated As of Cavanaugh (7.4.x), with no direct replacement
```

Comment:
Instead of "no direct replacement", lets point them to the mdn docs on the native method `setSelectionRange`

Reply:
I gave it a quick attempt but SF complained that it didn't follow a pattern